README.md

```diff
@@ -26,11 +26,11 @@ Hello, I´m Marco Díaz
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Portfolio part 5 of 10: Application deploy] <a name="about-project"></a>
+# 📖 [Portfolio part 6 of 10: Evaluate accessibility] <a name="about-project"></a>
 
-In this part of the project, I am using GitHub pages for the deployment of my web pages.
+**[Project 6: Evaluate accessibility**
```

Comment:
- [ ] Good job working on the accessibility 👍, but still need to to fix some points 😃. Kindly, make sure every image has `alt` with appropriate alternative text, that's useful for interacting or understanding the page has an `alt` text

![Capture2](https://user-images.githubusercontent.com/57408419/229758531-e052806f-6917-495b-88d3-ac84a315e013.JPG)
![Capture1](https://user-images.githubusercontent.com/57408419/229758599-ceb44b74-8733-43de-abc7-dcdde69bbd3f.JPG)
![alt](https://user-images.githubusercontent.com/57408419/229758624-9de95a7a-6960-4ef8-aaac-9df3da2561e0.JPG)